src/serializers/decoders/current/v11/decode_graph_schema.c

```diff
@@ -30,15 +30,17 @@ static Schema *_RdbLoadSchema(RedisModuleIO *rdb, SchemaType type) {
 
 		if(index_type == IDX_FULLTEXT) {
 			char *lang = RedisModule_LoadStringBuffer(rdb, NULL);
-			language = rm_strdup(lang);
+			if(!already_loaded) language = rm_strdup(lang);
 			RedisModule_Free(lang);
 			
 			uint stopwords_count = RedisModule_LoadUnsigned(rdb);
 			if(stopwords_count > 0) {
-				stopwords = array_new(char *, stopwords_count);
+				if(!already_loaded)
+					stopwords = array_new(char *, stopwords_count);
 				for (uint i = 0; i < stopwords_count; i++) {
 					char *stopword = RedisModule_LoadStringBuffer(rdb, NULL);
-					array_append(stopwords, rm_strdup(stopword));
+					if(!already_loaded)
+						array_append(stopwords, rm_strdup(stopword));
```

Comment:
I don't think this works; there's no place in which `stopwords` is retrieved in the `already_loaded` case - it is NULL.